politeiawww/api/www/v1/v1.go

```diff
@@ -750,6 +763,17 @@ type BatchProposalsReply struct {
 	Proposals []ProposalRecord `json:"proposals"`
 }
 
+// BatchVoteSummary is used to request the voting summary of multiple
+// proposals.
+type BatchVoteSummary struct {
+	Tokens []string `json:"tokens"`
+}
+
+// BatchVoteSummaryReply is used to reply to a BatchVoteSummary command.
+type BatchVoteSummaryReply struct {
```

Comment:
Would it make more sense to remove the `BestBlock` from the `VoteSummary` and add it to the `BatchVoteSummaryReply` instead?

Reply:
Yeah it does